website/docs/release-notes/2024-01-17-version-220.md

```diff
@@ -0,0 +1,50 @@
+---
+id: release-022
+title: AmigaOS 2.2 Dragon Fruit Release Notes
+authors: [mihelich, hackerman342, guilhermedemouraa]
+tags: [announcement, release]
+---
+
+We're excited to announce the launch of AmigaOS 2.2 “Dragon Fruit!” This is a
+usability-focused release improving the experience for large-scale data collection,
+custom application development, and teach-and-repeat autonomy using Autoplot.
+
+## Data Collection
+
+We've extended our suite of data collection apps with power features for large-scale
+collections.
+
+* **Camera**
+  * Improved UX for adjusting OAK-D camera settings.
+* **Recorder**
+  * Store user-defined metadata (key/value pairs) in log files, such as field name, crop
+    type, or whatever else is relevant to your collection needs!
+  * Split large logs into multiple chunks with a maximum file size for easier transfer.
+* **File Manager**
+  * Preview images and inspect metadata in log files.
+  * Support multiple USB drives.
+
+## Developer Experience
+
+* New and updated documentation for
+  [troubleshooting custom apps](/docs/brain/brain-apps#debugging-the-app-on-the-launcher),
+  the [microcontroller API](/docs/api/#farm-ng-microcontroller-adk), and debugging
+  [dashboard error codes](/docs/dashboard/dashboard-debugging).
+* More robust registration of custom apps with the launcher UI.
+* The custom app template now demonstrates using an app-specific virtualenv, so that
+  you can add additional Python packages without relying on system installs.
+  :::tip
+  As part of this change, the `python` manifest option has been removed, and you must
+  use `exec_cmd` to specify how to run your application. See the example
+  [manifest.json](https://github.com/farm-ng/amiga-app-template/blob/main/manifest.json).
+  :::
+
+## Autonomy
+
+* Autoplot allows deleting the last waypoint(s) while recording a track, so that you
+  can fix errors without starting the whole track over.
+* If the filter diverges, the reason is recorded in the `FilterState` proto.
+
```

Comment:
It is probably worth mentioning the modifications in the "Execute" button as well since it's somehow related to logging divergence reasons. If any of our autonomy requirements are not fulfilled, the button will be disabled, and there will be a little question mark on it. When clicked, a modal will pop up on the screen, displaying actionable insights to the user.
<img width="1728" alt="Screenshot 2024-01-18 at 10 20 50 AM" src="https://github.com/farm-ng/amiga-dev-kit/assets/39603677/f889183a-ab15-4235-a7db-ecad6e7ec7aa">
<img width="1728" alt="Screenshot 2024-01-18 at 10 20 54 AM" src="https://github.com/farm-ng/amiga-dev-kit/assets/39603677/2511994c-a8cf-4333-8c2a-9607c60e3f92">

Reply:
Good call out, thanks. I updated the text and added one of the screenshots.